erlang/crypto_tests/rsa_private

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
IT'S A SECRET DON'T LOOK

Reply:
not me you have to worry about 🤖🤖🤖

Reply:
wait until we forget and use this key in the actual game...